src/Compiler/Microsoft.AspNetCore.Razor.Language/src/MetadataCollection.Enumerator.cs

```diff
@@ -44,7 +44,7 @@ public void Reset()
             _current = default;
         }
 
-        public void Dispose()
+        public readonly void Dispose()
```

Comment:
This was added because there's an analyzer in the IDE that suggested it. I didn't consider it all that deeply. I just saw that it was technically correct and kept the change. :smile: